### PR TITLE
refac: Add more logging to domain test

### PR DESCRIPTION
**Pull request by johevemi**

### CODE REVIEW

File: source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs [hunk 1]
```diff
@@ -110,6 +110,11 @@ private async Task<bool> WaitForCalculationToComplete(Guid calculationId)
                 defaultDelay);
             stopwatch.Stop();
             Console.WriteLine($"LOOK AT ME: Calculation took {stopwatch.Elapsed} to complete for calculationId {calculationId}");
+            if (stopwatch.Elapsed >= TimeSpan.FromMinutes(15))
+            {
+                Console.WriteLine($"LOOK AT ME: Calculation with calculationId {calculationId} took more than 15 minutes to complete. And failed the test");
```
Comment (BjarkeMeier) [inline source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs:115]: What is it about that "LOOK AT ME" thing? 😄 
How about using Error?
```suggestion
                Console.Error.WriteLine($"Calculation with ID {calculationId} took more than 15 minutes to complete");
```

Comment (BjarkeMeier) [reply source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs:115]: Comment/question applies to all our current similar logging.

Comment (johevemi) [reply source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs:115]: will do

File: source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs [hunk 2]
```diff
@@ -109,7 +109,12 @@ private async Task<bool> WaitForCalculationToComplete(Guid calculationId)
                 defaultTimeout,
                 defaultDelay);
             stopwatch.Stop();
-            Console.WriteLine($"LOOK AT ME: Calculation took {stopwatch.Elapsed} to complete for calculationId {calculationId}");
+            Console.WriteLine($"Calculation took {stopwatch.Elapsed} to complete for calculationId {calculationId}");
+            if (stopwatch.Elapsed >= TimeSpan.FromMinutes(15))
+            {
+                await Console.Error.WriteLineAsync($"Calculation with ID {calculationId} took more than 15 minutes to complete");
```
Comment (BjarkeMeier) [inline source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs:115]: Why are you writing async?

**Pull request closed without merging**